extension/src/LanguageFunctions.ts

```diff
@@ -950,7 +950,7 @@ export function matchTranslationsFromTranslationMap(
         });
       } else {
         const match = matchMap.get(transUnit.source);
-        if (!isNullOrUndefined(match)) {
+        if (!(match === undefined)) {
```

Comment:
Is this the way to write this condition?!

What's wrong with below?
```suggestion
        if (match !== undefined) {
```

Reply:
Nothing! brain was probably already asleep. I remember thinking that it felt awkward with the perenthesis  :rofl: